client/app/application.coffee

```diff
@@ -25,8 +27,13 @@ class App extends Application
     - layout: the application layout view, rendered.
     ###
     initialize: ->
+        steps = require './steps/config'
         @on 'start', (options) =>
-            @router = new Router app: @
+
+            @onboarding = new Onboarding({}, steps)
+            @onboarding.onStepChanged (step) => @handleStepChanged(step)
```

Comment:
Here we are just dealing with native onboarding component, that we designed framework and lib agnostic. Such a simple thing as dealing with one or two events should not require a library in my opinion. The code is flexible enough to evolve to a library dependant logic in the future.